site/src/app/v2/une-question/components/ContentSection/ContentSection.tsx

```diff
@@ -69,6 +70,12 @@ export default function ContentSection({ categoriesWithArticles }: Props) {
                     setSelectedCategory(category);
                     setSelectedArticle(null);
                     replace(`${pathname}`);
+                    push([
+                      'trackEvent',
+                      'View FAQ',
+                      `Click on a category(${category.id})`,
```

Comment:
J'aurai juste mis "Clicked" sur le 3eme param. Puisqu'on a déjà la catégorie cliquée dans le 4eme param.

Reply:
Question de cohérence avec les autres tags que j'ai mis aussi.

Reply:
Dac, je met aussi le categoryId et articleId dans le 4e param

Reply:
https://matomo.org/faq/reports/the-anatomy-of-an-event/

Reply:
Feedback
https://github.com/betagouv/pass-sport/pull/110/commits/9b2c0af98c46be9b2465f9a05b6bf6fd95f96281